src/utils/generateTerms.ts

```diff
@@ -0,0 +1,12 @@
+// executed by node.js
```

Comment:
I think executables should not be placed in `src/` at all, usually in another top level dir called `bin/`